src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -261,7 +261,7 @@ export class OsdListComponent implements OnInit {
       },
       {
         prop: 'stats_history.in_bytes',
-        name: this.i18n('Writes bytes'),
+        name: this.i18n('Write bytes'),
```

Comment:
These two columns refer to the number of bytes read and written, thus I guess it should be `Bytes read` and `Bytes written`.